lib/src/model/hms_ios_pip_controller.dart

```diff
@@ -0,0 +1,187 @@
+import 'package:flutter/material.dart';
+import 'package:hmssdk_flutter/hmssdk_flutter.dart';
+import 'package:hmssdk_flutter/src/service/platform_service.dart';
+
+/// [HMSIOSPIPController] is used to setup and start the PIP in iOS. To know more visit [here](https://www.100ms.live/docs/flutter/v2/advanced-features/pip-mode).
+class HMSIOSPIPController {
+  static bool _isPIPSetupDone = false;
+
+  ///setup PIP is used to setup PIP in iOS devices.
+  ///**parameters**:
+  ///
+  ///**autoEnterPip** - Enable [autoEnterPip] will start pip mode automatically when app minimized.
+  ///
+  ///**aspectRatio** - Ratio for PIP window. For example: [16,9], [9,16] ,[1,1]
+  ///
+  ///`Note: Use [changeTrackPIP] function to change track in PIP window. Default track is local peer video track if available.`
+  ///
+  ///Refer [PIP mode guide here](https://www.100ms.live/docs/flutter/v2/advanced-features/pip-mode)
+  static Future<HMSException?> setup(bool? autoEnterPip,
+      {List<int>? aspectRatio,
+      ScaleType scaleType = ScaleType.SCALE_ASPECT_FILL,
+      Color backgroundColor = Colors.black}) async {
+    var result =
+        await PlatformService.invokeMethod(PlatformMethod.setupPIP, arguments: {
+      "auto_enter_pip": autoEnterPip ?? true,
+      "ratio": aspectRatio ?? [16, 9],
+      "scale_type": scaleType.value,
+      "color": [
+        backgroundColor.red,
+        backgroundColor.green,
+        backgroundColor.blue
+      ]
+    });
+    if (result != null) {
+      return HMSException.fromMap(result["error"]);
+    }
+    _isPIPSetupDone = true;
+    return null;
+  }
+
+  /// [start] is used to start PIP manually.
+  ///
+  /// `Note: [setupPIP] is required to call before calling [startPIP].`
+  ///
+  ///Refer [PIP mode guide here](https://www.100ms.live/docs/flutter/v2/advanced-features/pip-mode)
+  static void start() {
+    if (_isPIPSetupDone) {
+      PlatformService.invokeMethod(PlatformMethod.startPIP);
+    }
+  }
+
+  /// [stop] is used to stop PIP manually.
+  ///
+  /// `Note: [setupPIP] is required to call before calling [stopPIP].`
+  ///
+  ///Refer [PIP mode guide here](https://www.100ms.live/docs/flutter/v2/advanced-features/pip-mode)
+  static void stop() {
+    if (_isPIPSetupDone) {
+      PlatformService.invokeMethod(PlatformMethod.stopPIP);
+    }
+  }
+
+  ///[changeTrack] is used to change the track of PIP window.
+  ///
+  ///**Parameters**:
+  ///
+  ///**track** - [HMSVideoTrack] need to be passed for changing PIP window track.
+  ///
+  ///**aspectRatio** - Ratio for PIP window. For example: [16,9], [9,16] ,[1,1]
+  ///
+  /// `Note: [setupPIP] is required to call before calling [changeTrackPIP].`
+  ///
+  /// Refer [PIP mode guide here](https://www.100ms.live/docs/flutter/v2/advanced-features/pip-mode)
+  static Future<HMSException?> changeTrack(
```

Comment:
should be renamed to changeVideoTrack
```suggestion
  static Future<HMSException?> changeVideoTrack(
```